server/services/utils/checkPriceThenUpdateDb.js

```diff
@@ -0,0 +1,22 @@
+const axios = require("axios");
+
+const checkPriceChange = (currentPrice, fetchedPrice, productId, name) => {
+  console.log(`Product Name ${name}`);
+  console.log(`Current Price ${currentPrice}`);
+  console.log(`Fetched Price ${fetchedPrice}`);
+
+  if (currentPrice !== fetchedPrice) {
+    const body = {
+      currentPrice,
+      newPrice: fetchedPrice,
+      id: productId,
+      name,
+    };
+
+    return axios.post("http://localhost:3001/notification/price", body);
```

Comment:
This will return a promise since it isn't awaited. Is that intentional?
Also, if not too inconvenient, I would recommend refactoring the route for posting a notification route so that there is a function you can call from both here and in the route so the server doesn't end up making a call to itself. But maybe I am being too perfectionistic :)